server/src/main/java/org/opensearch/search/profile/ProfileResult.java

```diff
@@ -102,19 +165,29 @@ public ProfileResult(StreamInput in) throws IOException {
         this.type = in.readString();
         this.description = in.readString();
         this.nodeTime = in.readLong();
+        this.nodeStartTime = in.readLong();
```

Comment:
You need to add a version check for the writer and parser since older versions do not understand how to read or write these new fields.

Reply:
Hi @andrross, thank you for the comment. It's addressed now.